src/test/java/teammates/test/cases/action/StudentCourseJoinLegacyLinkTest.java

```diff
@@ -0,0 +1,59 @@
+package teammates.test.cases.action;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
+import teammates.logic.core.StudentsLogic;
+import teammates.ui.controller.Action;
+import teammates.ui.controller.StudentCourseJoinAction;
+
+/**
+ * SUT: {@link StudentCourseJoinAction}.
```

Comment:
You may want to add more information about the purpose of this test since it does not explain why StudentCourseJoinActionTest is insufficient.